### PR TITLE
Feature/customise results and submit options

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: molo/surveys/templates/surveys/molo_survey_page.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h4 class="surveys__intro">{{ page.intro }}</h4>
             {% endif %}
           </fieldset>
         {% endfor %}
-        <input type="submit" value="{% if is_intermediate_step %}{% trans 'Next Question' %}{% else %}{% trans 'Submit Survey' %}{% endif %}" />
+        <input type="submit" value="{% if is_intermediate_step %}{% trans 'Next Question' %}{% else %}{{self.submit_text}}{% endif %}" />
```
Comment (saeedmb8) [inline molo/surveys/templates/surveys/molo_survey_page.html]: @codiebeulaine let's add the exciting text as a default value here, like this 
{% trans "Submit Survey" as text %}
`{{self.submit_text|default:text }}`

File: molo/surveys/templates/surveys/molo_survey_page_landing.html [hunk 2]
```diff
@@ -18,7 +18,7 @@ <h4 class="surveys__intro">
 		<h3 class="surveys__question">{{ question }}</h3>
 		{% for answer, count in answers.items %}
 		  <div class="surveys__answer">
-			<span>{{ answer }}</span> {{ count }}
+			<span>{{ answer }}</span> {% if self.show_results_as_percentage%}{{ count }}%{% else %}{{count}}{% endif %}
```
Comment (saeedmb8) [inline molo/surveys/templates/surveys/molo_survey_page_landing.html]: @codiebeulaine why the variable names are same here?

Comment (codiebeulaine) [inline molo/surveys/templates/surveys/molo_survey_page_landing.html]: @saeedmb8 the count is either given back as a percentage or as a total, but the first count has a percentage sign next to it

**Comment by codiebeulaine** [comment]
Thanks @saeedmb8 for in office 👍

**Pull request merged**